src/assets/icons/CityBikeIcon.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+function CityBikeIcon(): JSX.Element {
+    return (
+        <svg
+            width="24"
+            height="24"
+            viewBox="0 0 24 24"
+            fill="white"
```

Comment:
Tenker currentColor kan gi mening her for mulig styling med andre farger

Reply:
Yep